# This represents a timestamp with a reference header, incl. a reference timestamp
Header header
time time